easybuild/easyconfigs/j/JupyterHub/JupyterHub-1.4.1-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,117 @@
+easyblock = 'PythonBundle'
+
+name = 'JupyterHub'
+version = '1.4.1'
+
+homepage = 'https://jupyter.org'
+description = """JupyterHub is a multiuser version of the Jupyter (IPython) notebook designed
+ for centralized deployments in companies, university classrooms and research labs."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+builddependencies = {
+    ('binutils', '2.36.1'),
+}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('IPython', '7.25.0'),
+    ('Mako', '1.1.4'),
+    ('configurable-http-proxy', '4.4.0'),
+    ('OpenSSL', '1.1', '', True),
+]
+
+use_pip = True
+
+exts_list = [
+    ('certipy', '0.1.3', {
+        'checksums': ['695704b7716b033375c9a1324d0d30f27110a28895c40151a90ec07ff1032859'],
+    }),
+    ('pamela', '1.0.0', {
+        'checksums': ['65c9389bef7d1bb0b168813b6be21964df32016923aac7515bdf05366acbab6c'],
+    }),
+    ('greenlet', '1.1.0', {
+        'checksums': ['c87df8ae3f01ffb4483c796fe1b15232ce2b219f0b18126948616224d3f658ee'],
+    }),
+    ('SQLAlchemy', '1.4.20', {
+        'checksums': ['38ee3a266afef2978e82824650457f70c5d74ec0cadec1b10fe5ed6f038eb5d0'],
+    }),
+    ('alembic', '1.6.5', {
+        'checksums': ['a21fedebb3fb8f6bbbba51a11114f08c78709377051384c9c5ead5705ee93a51'],
+    }),
+    ('oauthlib', '3.1.1', {
+        'checksums': ['8f0215fcc533dd8dd1bee6f4c412d4f0cd7297307d43ac61666389e3bc3198a3'],
+    }),
+    ('pyOpenSSL', '20.0.1', {
+        'modulename': 'OpenSSL',
+        'checksums': ['4c231c759543ba02560fcd2480c48dcec4dae34c9da7d3747c508227e0624b51'],
+    }),
+    ('python-json-logger', '2.0.1', {
+        'modulename': 'pythonjsonlogger',
+        'checksums': ['f26eea7898db40609563bed0a7ca11af12e2a79858632706d835a0f961b7d398'],
+    }),
+    ('ruamel.yaml', '0.17.10', {
+        'checksums': ['106bc8d6dc6a0ff7c9196a47570432036f41d556b779c6b4e618085f57e39e67'],
+    }),
+    ('ruamel.yaml.clib', '0.2.4', {
+        'modulename': False,
+        'checksums': ['f997f13fd94e37e8b7d7dbe759088bb428adc6570da06b64a913d932d891ac8d'],
+    }),
+    ('python-editor', '1.0.4', {
+        'modulename': 'editor',
+        'checksums': ['51fda6bcc5ddbbb7063b2af7509e43bd84bfc32a4ff71349ec7847713882327b'],
+    }),
+    ('jupyter-telemetry', '0.1.0', {
+        'source_tmpl': 'jupyter_telemetry-%(version)s.tar.gz',
+        'checksums': ['445c613ae3df70d255fe3de202f936bba8b77b4055c43207edf22468ac875314'],
+    }),
+    ('jupyterhub', version, {
+        'checksums': ['ee1b0718a4db8e0b339796e3e50b704ca6822ab22a7435289dbb5932f65b5199'],
+    }),
+    ('batchspawner', '1.1.0', {
+        'checksums': ['11fe187844e311531597cde841660b4d949f1131726d3701c57ba6080ba15c1a'],
```

Comment:
@Micket in my build attempts it finds `9bae72f7c1bd9bb11aa58ecc3bc9fae5475a10fdd92dc0c0d67fa7eb95c9dd3a` as the checksum.

Reply:
We used to use the old sources from gjthub, but, it turned out they finally added it to pypi.org so i changed it. I'm not sure they are functionally identical. So.. that's annoying since the old jupyterhub's sources will be the same. We can update both to use pypi, and/or add both checksums here.

Reply:
I was wrong, we haven't used batchspawner 1.1.0 before, so the solution is simple. Just fix the checksum here.